src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,25 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String bladesTypes;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, String bladesTypes) {
+        setName(name);
+        setColor(color);
```

Comment:
use a constructor of the parent to initialize these fields, the same for other figures